tests/commands/run_test.py

```diff
@@ -633,6 +633,30 @@ def _run_for_stage(stage):
     assert _run_for_stage('prepare-commit-msg').startswith(b'hook 4...')
     assert _run_for_stage('commit-msg').startswith(b'hook 5...')
 
+def test_push_remote_environment(cap_out, store, repo_with_passing_hook):
+    config = {
+        'repo': 'local',
+        'hooks': [
+            {
+                'id': 'print-push-remote',
+                'name': 'Print push remote name',
+                'entry': 'python -c \'import os; print(os.environ["PRE_PUSH_REMOTE_NAME"])\'',
+                'language': 'system',
```

Comment:
a (sadly) more portable alternative is to use bash here:

```
entry: bash -c 'echo "$PRE_PUSH_REMOTE_NAME"'
```

the main reason being the tests run in some environments where `python` doesn't exist (only `python3`) -- and then if you use `python3` then it doesn't work on windows which only has `python` :man_facepalming: